src/styles/blocks/watch.scss

```diff
@@ -0,0 +1,48 @@
+.watch {
+  @include circle($widthClock);
+
+  border: 1vmin dotted $colorBorder;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  &::before {
+    @include after-before($widthSecond, $heightSecond, $colorSecond, 2);
+
+    animation-duration: $stepSecond;
+    animation-timing-function: linear;
+  }
+
+  &::after {
+    @include after-before($widthMinute, $heightMinute, $colorMinute, 1);
+
+    animation-duration: $stepSecond * 60;
+    animation-timing-function: steps(60, end);
+  }
```

Comment:
Same applies to the `::after` selector. Adding a comment to explain its purpose would be helpful.